Robotics/Apps section remove intermediate ROS abstraction (#3358)
PX4/PX4-user_guide: en/ros/offboard_control.md

> # Offboard Control 

Author: FYI this doc is first doc in the  Robotics API section. It is supposed to explain how offboard control works/what it is.

The concept is useful but this is so out of date it probably should be deleted. This is added to make sure I look at this topic next week and decide what can be used or to discard it.

### Map an RC switch to offboard mode activation

> To do this, load up the parameters in _QGroundControl_ and look for the RC_MAP_OFFB_SW parameter to which you can assign the RC channel you want to use to activate offboard mode.

Author: Thanks. Slight mod.

```suggestion
In _QGroundControl_ you can modify the [RC_MAP_OFFB_SW](../advanced_config/parameter_reference.md#RC_MAP_OFFB_SW) parameter to set the RC channel that will be used to activate offboard mode.
```

> It can be useful to map things in such a way that when you fall out of offboard mode you go into position control.

Author: @beniaminopozzan This doesn't make much sense - why does this help you go into position control when you fall out of offboard mode? All it does is say that you can enable/disable offboard mode?
Or are you saying this is an easy way to disable offboard mode if needed, and by default you'll fall into position mode?

Reviewer: @hamishwillee , it is indeed a bit misleading.
The RC switch allows you to enter and exit offboard mode, when lo leave it you'll return to the mode set by the mode switch in the RC which is not necessary position mode.
So line 18 should be "It can be useful to map things in such a way that when you fall out of offboard mode you go into **manual** control."
An even better approach is to use https://docs.px4.io/main/en/advanced_config/parameter_reference.html#COM_RC_OVERRIDE so the user just need to move the sticks to exit offboad though.

Author: Thank you! See https://github.com/PX4/PX4-user_guide/pull/3372

> Although this step isn't mandatory since you can activate offboard mode using a MAVLink message.

Author: @beniaminopozzan I think you're saying here that you don't need to map a switch because you can do so in other ways? I.e. using GCS or MAVLink? 

Depending on your answer I think I'll modify this as a post process.

Reviewer: @hamishwillee I was not saying that but you are right, you can enter offboard mode via GCS, via MAVLink command, via ROS 2 and of course via RC.
